src/hooks/useTokenLauncher.ts

```diff
@@ -15,15 +15,10 @@ TL.configure(config);
 export function useTokenLauncher(): typeof TokenLauncher {
   const { getTokenLauncherSupportedChainInfo } = useBackendNetworksStore();
   const launcherSupportedChains = getTokenLauncherSupportedChainInfo();
```

Comment:
Any reason this isn't using a selector?

```ts
const launcherSupportedChains = useBackendNetworksStore(state => state.getTokenLauncherSupportedChainInfo());
```

Reply:
Nope. Haven't had a chance to review my changes and polish syntax yet. This hook probably won't exist by tomorrow.